### ISSUE TITLE
detail training setting

### ISSUE THREAD

**Issue by GYxiaOH** [ericsun99/ShuffleNet-1g8-Pytorch#1]
Hello,can you share your training log for me or more detail training setting? i can't replicate the paper result but i use the same model file with you

**Comment by lxtGH** [comment]
Hi，can you provide the training details ???
